scheduler/src/test/scala/com/sky/kms/e2e/SchedulerIntSpec.scala

```diff
@@ -16,8 +16,9 @@ class SchedulerIntSpec extends SchedulerIntSpecBase {
         withSchedulerApp {
           val schedules = createSchedules(2, forTopics = List(scheduleTopic, extraScheduleTopic))
 
-          publish(schedules) andThen
-            (assertMessagesWrittenFrom(_, schedules))
+          publish(schedules)
+            .foreach(assertMessagesWrittenFrom(_, schedules))
```

Comment:
The `andThen` returns a partial function which was never actually called, so the assertion never got executed.